content/it/partners/_index.html

```diff
@@ -1,135 +1,43 @@
 ---
-draft: True
-title: Partners
-bigheader: Kubernetes Partners
-abstract: Growing the Kubernetes ecosystem.
+title: Partner
+bigheader: I partner di Kubernetes
+abstract: Far crescere l'ecosistema di Kubernetes.
 class: gridPage
 cid: partners
 ---
 
 <section id="users">
-    <main>
-        <h5>Kubernetes collabora con i partner per creare una base di codice forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
-        <div class="col-container">
-          <div class="col-nav">
-            <center>
-              <h5---
-title: Partners
-bigheader: I Partners di Kubernetes
-abstract: La Crescita dell'ecosistema Kubernetes.
-class: gridPage
-cid: partners
----
-<meta charset="utf-8" />
-
-<section id="users">
-    <main>
-        <h5>Kubernetes lavora in collaborazione con diversi partner al fine di creare un codice dalla base forte e dinamica che supporti uno spettro di piattaforme complementari.</h5>
+    <main class="main-section">
+        <h5>Kubernetes collabora con i partner per creare un sorgente forte e vivace che supporti uno spettro di piattaforme complementari.</h5>
         <div class="col-container">
           <div class="col-nav">
             <center>
               <h5>
-                <b>Fornitori di Servizi Certificati Kubernetes</b>
+                <b>Fornitori Certificati di Servizi su Kubernetes</b>
               </h5>
-              <br>Fornitori di servizi controllati con grande esperienza nell'aiutare le imprese ad adottare con successo Kubernetes.
+              <br>Fornitori verificati di servizi con grande esperienza nell'aiutare le imprese ad adottare con successo Kubernetes.
               <br><br><br>
-              <button id="kcsp" class="button" onClick="updateSrc(this.id)">Guarda i Partners di KCSP</button>
-              <br><br>Interessato a diventare partner <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>?
+              <button id="kcsp" class="button" onClick="updateSrc(this.id)">Guarda i Partners KCSP</button>
+              <br><br>Interessato a diventare un partner <a href="https://www.cncf.io/certification/kcsp/">KCSP</a>?
             </center>
           </div>
           <div class="col-nav">
             <center>
               <h5>
-                <b>Distribuzioni Kubernetes Certificate, Piattaforme Hosted e Installers</b>
-              </h5>La conformità del software assicura che la versione di Kubernetes di ogni fornitore supporti le API richieste.
+                <b>Distribuzioni Kubernetes Certificate, Piattaforme Hosted e Software di installazione</b>
```

Comment:
what about "Distribuzioni di Kubernetes Certificate, Certified Hosted Platforms and Software di installazione Certificati"